coffee_core/src/coffee.rs

```diff
@@ -281,6 +282,21 @@ impl PluginManager for CoffeeManager {
         self.storage.store(&self.storage_info()).await?;
         Ok(())
     }
+
+    async fn remove_remote(&mut self, name: &str) -> Result<(), CoffeeError> {
+        debug!("remote removing: {}", name);
+        let index = self
+            .repos
+            .iter()
+            .position(|x| &*x.to_owned().name() == name)
+            .unwrap();
+        let repo_path = &self.repos[index].url().path_string;
+        fs::remove_dir_all(repo_path).unwrap();
```

Comment:
```suggestion
        fs::remove_dir_all(repo_path)?;
```

or you can use the `map_err(|err| CoffeError::new(1, &format!("{err}"))?`

The unwrap should be never called inside a function call because you are breaking the normal life cycle of an application.